Web/src/components/results-leaderboard/Results.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ChevronDown, Icon } from 'react-feather';
+import { Container, Group } from './Containers';
+
+interface ResultsProps {
+  score:number,
+  gameText:string,
+  rank:number,
+  numPlayers:number,
+}
+
+// TODO: Use secondary font instead of primary font
+// Secondary font's height is too tall when rendering on mobile devices.
```

Comment:
Hmm try setting margin to 0?

Reply:
This solved it on mobile! But on web it seems like on web the number sits too low now 😂 I think it's fine because our games are mobile anyway!
![image](https://user-images.githubusercontent.com/20502803/68049894-f7380280-fd1e-11e9-95ce-c67193d60354.png)